app/components/form/fields/TextField.tsx

```diff
@@ -30,10 +30,8 @@ import { ErrorMessage } from './ErrorMessage'
 export interface TextFieldProps<
   TFieldValues extends FieldValues,
   TName extends FieldPath<TFieldValues>,
-> extends UITextFieldProps {
+> extends Omit<UITextFieldProps, 'type'> {
   name: TName
-  /** HTML type attribute, defaults to text */
-  type?: string
```

Comment:
This was necessary when we needed to differentiate between the `text` and `number` uses of TextField. Now that numbers will be going through NumberField, this prop isn't necessary.

Reply:
Good catch!